screens/signUp.js

```diff
@@ -1,11 +1,16 @@
+/* eslint-disable no-unused-vars */
+import { useState } from "react";
 import { Image, View, Text, TouchableOpacity } from "react-native";
 import { useTailwind } from "tailwind-rn";
 
 import { IconInput } from "../components/inputs";
-import { RoundButton } from "../components/buttons";
 
 const SignUp = () => {
   const tailwind = useTailwind();
+  const [email, setEmail] = useState("");
+  const [name, setName] = useState("");
+  const [password, setPassword] = useState("");
+  const [dbInsert, setDbInsert] = useState([]);
```

Comment:
It looks like you are not using `dbInsert` anywhere in the code, so there is no need to create a state for it.